app/post_metadata.go

```diff
@@ -422,12 +422,7 @@ func (a *App) getLinkMetadata(requestURL string, timestamp int64, isNewPost bool
 
 		client := a.HTTPService().MakeClient(false)
 		client.Timeout = time.Duration(*a.Config().ExperimentalSettings.LinkMetadataTimeoutMilliseconds) * time.Millisecond
-		mmTransport := a.HTTPService().MakeTransport(false)
-		client.Transport = mmTransport.Transport
-
-		if strings.HasPrefix(requestURL, "https://twitter.com/") || strings.HasPrefix(requestURL, "https://mobile.twitter.com/") {
-			request.Header.Add("User-Agent", "facebookexternalhit/1.1 (+http://www.facebook.com/externalhit_uatext.php)")
-		}
+		client.Transport = a.HTTPService().MakeTransport(false)
```

Comment:
I don't think this line should be necessary any more. I think `MakeClient` sets `client.Transport` to `MakeTransport()` by default

Reply:
Good catch! Removing.